lib/web/desktop.go

```diff
@@ -79,8 +116,12 @@ func (h *Handler) handleDesktopAccessWebsocket(
 	}
 	defer serviceCon.Close()
 	tlsConfig := ctx.clt.Config()
-	// Pass target desktop UUID via SNI.
-	tlsConfig.ServerName = desktopName + desktop.SNISuffix
+	// Pass target parameters via SNI: "login_width_height_desktopName.SNISuffix".
+	tlsConfig.ServerName = strings.Join([]string{
```

Comment:
I don't love adding to the server name here. The server name should really just be used to identify the server we're connecting to.

Could we do away with this, and just send the TDP messages with login and screen size ourselves before we start proxying TDP messages from the browser?